From a383150faf514d1244a9ee6ad86b2589d4a8ae5e Mon Sep 17 00:00:00 2001
From: Pedro Palazon Candel <pedro@joyent.com>
Date: Mon, 6 May 2019 17:43:22 +0200
Subject: [PATCH] TRITON-1299 need logarchiver-agent service to run along side
 hermes-actor Reviewed by: Mike Gerdts <mike.gerdts@joyent.com> Approved by:
 Mike Gerdts <mike.gerdts@joyent.com>

---
 deps/hermes | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/hermes b/deps/hermes
index f39da64..73063d9 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit f39da64fb3ce370e0f3b24d8d96a9c5e6ecd0a79
+Subproject commit 73063d9fe9399b927c621df16e473503f373e0fe
-- 
2.21.0

